server/config.go

```diff
@@ -224,36 +221,13 @@ func (cf *Config) GetChainstore() (chainstore.Store, error) {
 	return store, nil
 }
 
-// Setup app stats & instrumentation
-func (cf *Config) SetupLibrato() (err error) {
-
-	if cf.Librato.Enabled {
-		reporter := librato.NewReporter(
-			metrics.DefaultRegistry,
-			10e9,
-			cf.Librato.Email,
-			cf.Librato.Token,
-			cf.Librato.Source,
-			[]float64{0.5, 0.95},
-			time.Millisecond,
-		)
-		reporter.Namespace = cf.Librato.Namespace
-		go reporter.Run()
-
-		// TODO: should we add "pulse" gauge metric to 1 ...? ...... or as a counter.....?
-
-		// TODO: is there an easy way to offer sys.cpu sys.mem sys.swap stats..?
-		// for convienience..
-
-		// go metrics.Log(metrics.DefaultRegistry, 10e9, log.New(os.Stderr, "metrics: ", log.Lmicroseconds))
-
-		// Capture Runtime stats periodically.
-		// NOTE: reading the MemStats will stop the world, so do this every > 1min
-		metrics.RegisterRuntimeMemStats(metrics.DefaultRegistry)
-		go metrics.CaptureRuntimeMemStats(metrics.DefaultRegistry, 60e9)
-	} else {
-		metrics.UseNilMetrics = true
+func (cf *Config) SetupStatsD() error {
+	if cf.StatsD.Enabled {
+		sink, err := metrics.NewStatsdSink(cf.StatsD.Address)
```

Comment:
does the new metrics library have anything for capturing the Go's runtime stats like the stuff that runtime/debug will report? https://golang.org/pkg/runtime/#pkg-index  .. things like num of goroutines, num of cgo calls, gc pauses, etc.

the old metrics library did this with `go metrics.CaptureRuntimeMemStats(metrics.DefaultRegistry, 60e9)` .. perhaps, armon's metrics lib has something similar? if not, lets add it :)